vue-app/src/components/CartWidget.vue

```diff
@@ -12,14 +12,15 @@
       @click="toggleCart"
     >
       <img alt="open" width="16px" src="@/assets/chevron-left.svg" />
-      <div
-        :class="[
-          cart.length + -0 ? 'circle pulse cart-indicator' : 'cart-indicator',
-        ]"
-        v-if="!$store.state.showCartPanel && isCartBadgeShown"
-      >
-        {{ cart.length }}
-      </div>
+      <transition name="pulse" mode="out-in">
+        <div
+          :key="cart.length"
```

Comment:
Using `<transition>` whenever the `key` changes, it's going to execute the `pulse` animation. More on this here: https://vuejs.org/v2/guide/transitions.html#Transitioning-Between-Elements